src/lib/components/chat/ChatWindow.svelte

```diff
@@ -47,7 +53,25 @@
 		message = "";
 	};
 
+	let lastTarget: EventTarget | null = null;
+
+	let onDrag = false;
+
+	const onDragEnter = (e: DragEvent) => {
+		lastTarget = e.target;
+		onDrag = true;
```

Comment:
two comment on the `FileDropZone`:
1. when `onDrag=true`, there should be some kind of visual feedback (like the background changed to different color etc.)
2. At the moment, the `FileDropZone` only works if you drag/drop file on the `Upload image` btn. It may make sense to have a similar behaviour as ChatGPT, where `FileDropZone` works on the wntire window, not just `Upload image` btn

Reply:
I think [we'll handle that in a separate PR](https://github.com/huggingface/chat-ui/pull/531#pullrequestreview-1731936802) but definitely good points